### ISSUE TITLE
Formatting Decision - current test table format in Metworx requirement spec does not match official template

### ISSUE THREAD

**Issue by tkwilos** [metrumresearchgroup/mrgvalidate#96]
This will warrant further discussion amongst Seth G, Mike C, and maybe others. 

Currently, the table listing the tests in the Metworx Requirements Specification document does not match the format on our official, QA reviewed/approved template. 

For reference, here is the version presently produced via `mrgvalidate`:
<img width="654" alt="image" src="https://user-images.githubusercontent.com/67168516/206863652-183fede7-9765-4e81-b77a-2ca6915cd6e5.png">

And here is the version on the current version of the template:
<img width="654" alt="image" src="https://user-images.githubusercontent.com/67168516/206863700-2ee55a4c-2cb1-45f0-9f07-799e680f8dac.png">

**Comment by seth127** [comment]
@tkwilos are you literally talking about the formatting (font, size, color, etc.)? I think the content looks identical, right?

**Comment by barrettk** [comment]
Yeah we changed the format when we changed how the docs get generated. I thought we had sent updated templates, but regardless the new format is the standard.

**Comment by tkwilos** [comment]
@seth127 yes sorry I'm talking about the formatting, not the content itself. 

It seems then, potentially I'm out of sync with the expected format of these tables...looking at the template Mike C sent me for the Metworx management portal 4.0 validation and comparing it to the docs produced here, we're not quite aligned.
